easybuild/easyconfigs/k/kokkos/kokkos-3.4.01-GCC-10.3.0.eb

```diff
@@ -0,0 +1,43 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'CMakeMake'
+
+name = 'kokkos'
+version = '3.4.01'
+
+homepage = 'https://kokkos.org'
+description = """Kokkos Core implements a programming model in C++ for writing
+                 performance portable applications targeting all major HPC
```

Comment:
Please strip leading spaces to a single one, or the output of `module show` is going to look weird...

Reply:
I did.